### PR TITLE
#168052207 Helper for creating JWT

### PR DESCRIPTION
**Pull request by dieudonneAwa**
**What does this PR do?**
- Create json web token for user registration

**Description of Task to be completed?**
- Create a file to process tokens
- This file exports a class called ProcessToken that has two helper methods i.e `createToken` and `verifyToken`
- Import this file and destructure out the helper method you want to use
- `createToken` will create token by passing a user object as the payload.
- 'verifyToken` will verify the token 

**How should this be manually tested?**
After cloning the repo,
- run `npm i` to install `jsonwebtoken` dependency.
- import the file whenever you want to use and and destructure the class methods from it
- pass in the user object from which you want to create the token
- possibly store the returned value in a variable called `token`
- you can now use the token 

**Any background context you want to provide?**
- [JSON WEB TOKEN](https://www.npmjs.com/package/jsonwebtoken)

 **What are the relevant stories?**
[#168052207](https://www.pivotaltracker.com/n/projects/2381984/stories/168052207)

**Screenshots (if appropriate)**
- None

**Questions:**
- None

### CODE REVIEW

File: src/utils/createToken.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import jwt from 'jsonwebtoken';
+import { config } from 'dotenv';
+
+config();
+
+export default (user) => {
+  const token = jwt.sign({
+    id: user.id,
```
Comment (Abobos) [inline src/utils/createToken.js]: Well done, In your token function you are retrieving the user_id and user_email.. can you make it flexible in a such way that we could pass a payload(an object).. reason being that we could want to encode other data let alone the user's id and email.

**Review by Wokoro — APPROVED** [review]
Please document how this method can be used for clearity....

**Comment by dieudonneAwa** [comment]
> Please document how this method can be used for clearity....

I did that already in the PR Description @Wokoro

**Review by Kingobaino — COMMENTED** [review]
It is cool but I am wondering why you have to do the long maths in expiration period when you can easily write the time in hours or miliseconds

**Comment by dieudonneAwa** [comment]
> It is cool but I am wondering why you have to do the long maths in expiration period when you can easily write the time in hours or miliseconds

I have resolved that @Kingobaino

**Review by richienabuk — APPROVED** [review]
LGTM

**Review by Kingobaino — APPROVED** [review]

**Pull request closed without merging**